sdk/keyvault/azure-security-keyvault-keys/test/ut/CMakeLists.txt

```diff
@@ -37,15 +38,15 @@ gtest_discover_tests(azure-security-keyvault-keys-test
 ################## Live Tests ##########################
 add_executable (
   azure-security-keyvault-keys-test-live
-  key_client_backup_test_live.cpp
-  key_client_create_test_live.cpp
-  key_client_delete_test_live.cpp
-  key_client_get_test_live.cpp
-  key_client_import_test_live.cpp
-  key_client_update_test_live.cpp
-  key_cryptographic_client_test_live.cpp
-  main.cpp
-  key_client_base_test.hpp
+    azure_security_keyvault_keys_test.cpp
+    key_client_backup_test_live.cpp
+    key_client_create_test_live.cpp
```

Comment:
I think, it should be `live_test` instead of `test_live`, for consistency, plus it reads better